test/tests/functional/pbs_cpuset.py

```diff
@@ -151,7 +152,7 @@ def test_reliable_job_startup_on_cpuset(self):
 
         # Check for msg in mom superior logs
```

Comment:
update the comment to say what you are doing and why

Reply:
Done